Atomic/util.py

```diff
@@ -63,22 +63,61 @@ def get_docker_conf():
         dconf = c.info()
     return dconf
 
+
+def registries_tool_path():
+    if os.path.exists("/usr/libexec/registries"):
```

Comment:
Can you make this a constant that is available via /etc/atomic.conf.   Would like to see us able to change this on the fly especially for testing.